js/digifinex.js

```diff
@@ -1445,6 +1446,63 @@ module.exports = class digifinex extends Exchange {
         };
     }
 
+    parseTransfer (transfer, currency = undefined) {
+        //
+        //     {
+        //         "code": 0
+        //     }
+        //
+        return {
+            'info': transfer,
+            'id': undefined,
+            'timestamp': undefined,
+            'datetime': undefined,
+            'currency': this.safeCurrencyCode (undefined, currency),
+            'amount': this.safeNumber (transfer, 'amount'),
+            'fromAccount': this.safeString (transfer, 'fromAccount'),
+            'toAccount': this.safeString (transfer, 'toAccount'),
+            'status': this.safeInteger (transfer, 'code'),
+        };
+    }
+
+    async transfer (code, amount, fromAccount, toAccount, params = {}) {
+        await this.loadMarkets ();
+        const currency = this.currency (code);
+        if (fromAccount === 'spot') {
+            fromAccount = 1;
+        } else if (fromAccount === 'margin') {
+            fromAccount = 2;
+        } else {
+            fromAccount = 3; // OTC account
+        }
+        if (toAccount === 'spot') {
+            toAccount = 1;
+        } else if (toAccount === 'margin') {
+            toAccount = 2;
+        } else {
+            toAccount = 3; // OTC account
+        }
```

Comment:
Let's rewrite it slightly / copy this bit from here:

- https://github.com/ccxt/ccxt/blob/master/js/okx.js#L3842
- https://github.com/ccxt/ccxt/blob/master/js/okx.js#L640